### PR TITLE
Fix ERROR: get_child: Index p_index out of size

### LINKED ISSUE #12189: ERROR: get_child: Index p_index out of size (data.children.size()).
**Operating system or device, Godot version, GPU Model and driver (if graphics related):**
Godot 2.1.x editor, custom build, target=release_debug (no errors in 2.1.4 stable build)
Linux Mint 18.2

**Issue description:**
ERROR: connect: Condition ' s->slot_map.has(target) ' is true. returned: ERR_INVALID_PARAMETER
   At: core/object.cpp:1367.
ERROR: get_child: Index p_index out of size (data.children.size()).
   At: scene/main/node.cpp:821.

Second error repeats "randomly". Every time I move editor window or do something like adding new nodes, it is printed two times.

**Steps to reproduce:**
Just create new empty project.

### PR DESCRIPTION
**Pull request by Listwon**
Fixes #12189

### CODE REVIEW

File: editor/plugins/script_editor_plugin.cpp [hunk 1]
```diff
@@ -1924,16 +1924,18 @@ void ScriptEditor::_update_members_overview_visibility() {
 void ScriptEditor::_update_members_overview() {
 	members_overview->clear();
 
-	Node *current = tab_container->get_child(tab_container->get_current_tab());
-	ScriptTextEditor *ste = tab_container->get_child(tab_container->get_current_tab())->cast_to<ScriptTextEditor>();
-	if (!ste) {
-		return;
-	}
+	if(tab_container->get_child_count() > 0) {
```
Comment (akien-mga) [inline editor/plugins/script_editor_plugin.cpp]: `clang-format` will complain about the style here, see http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

**Comment by akien-mga** [comment]
CC @Paulb23, I wonder if this is the right fix. If this method makes no sense when `tab_container` is empty, it should likely never be called in that situation, instead of checking a posteriori.

File: editor/plugins/script_editor_plugin.cpp (hunk 1 above)
Comment (Listwon) [reply editor/plugins/script_editor_plugin.cpp]: Fixed.

**Comment by Listwon** [comment]
@akien-mga I think you are right. Sorry, I spent last two days just looking for the culprit, I'll appreciate your suggestions for better solution (this method is called in two places, so we need two checks if I'm not mistaken).

**Comment by Paulb23** [comment]
Apologies just saw this one - Looks like I forgot to port some of the code over when making the 2.1 version:
* [2.1](https://github.com/godotengine/godot/blob/2.1/editor/plugins/script_editor_plugin.cpp#L1916)
* [master](https://github.com/godotengine/godot/blob/master/editor/plugins/script_editor_plugin.cpp#L1377) 

The two functions:
```
_update_members_overview_visibility()
_update_members_overview()
```
Should be guarded with:
```
int selected = tab_container->get_current_tab();
if (selected < 0 || selected >= tab_container->get_child_count())
	return;

```

**Comment by akien-mga** [comment]
@Paulb23 Could you make a PR with those missed changes?

Closing this one in the meantime as it's not the correct fix. (But thanks for the proposal, it helped narrow down what actually needs to be done).

**Pull request closed without merging**